server/plugin/template.go

```diff
@@ -183,13 +183,39 @@ Assignees: {{range $i, $el := .Assignees -}} {{- if $i}}, {{end}}{{template "use
 {{- end -}}
 `))
 
-	template.Must(masterTemplate.New("newPR").Funcs(funcMap).Parse(`
+	template.Must(masterTemplate.New("newDraftPR").Funcs(funcMap).Parse(`
 {{ if eq .Config.Style "collapsed" -}}
 {{template "repo" .Event.GetRepo}} New pull request {{template "pullRequest" .Event.GetPullRequest}} was opened by {{template "user" .Event.GetSender}}.
 {{- else -}}
 #### {{.Event.GetPullRequest.GetTitle}}
 ##### {{template "eventRepoPullRequest" .Event}}
 #new-pull-request by {{template "user" .Event.GetSender}}
```

Comment:
Per https://github.com/mattermost/mattermost-plugin-github/issues/680#issuecomment-1576257866 this should always render a one-line notification, i.e. the collapsed style should be used.